src/helics/core/BrokerBase.cpp

```diff
@@ -150,6 +151,24 @@ std::shared_ptr<helicsCLI11App> BrokerBase::generateBaseCLI()
         "--globaltime",
         globalTime,
         "specify that the broker should use a globalTime coordinator to coordinate a master clock time with all federates");
+    hApp->add_flag(
+        "--async",
+        asyncTime,
+        "specify that the federation should use the asynchronous time coordinator (only minimal time management is handled in HELICS and federates are allowed to operate independently)");
+    hApp->add_option_function<std::string>(
+            "--timing",
+            [this](const std::string& arg) {
+                if (arg == "async") {
+                    asyncTime = true;
+                } else if (arg == "global") {
+                    globalTime = true;
+                } else {
+                    asyncTime = false;
+                    globalTime = false;
+                }
+            },
+            "specify the timing method to use in the broker")
+        ->check(CLI::IsMember({"async", "global", "distributed"}));
```

Comment:
```suggestion
        ->check(CLI::IsMember({"async", "global", "distributed", "default"}));
```

Documentation lists "default" as one of the values.